### ISSUE TITLE
More events

### ISSUE THREAD

**Issue by ajb** [dobtco/formbuilder#7]

**Comment by RUPOJS** [comment]
@adamjacobbecker can u give more details??

**Comment by ajb** [comment]
Yes -- a developer should be able to listen for certain events so that they can easily customize Formbuilder without overriding its default methods. 

Let's leave this issue as a place to discuss which events should be added -- right now the only one is `form.on 'save'`

**Comment by gladson** [comment]
I would like to see an event, form.on 'click_save'.
For that it is not saving automatically, allowing the User choice.

**Comment by Jargonius** [comment]
I'd love to see an event that triggers an update so the save button enables again.
